docs/apache-airflow/production-deployment.rst

```diff
@@ -749,6 +749,105 @@ additional apt dev and runtime dependencies.
+production, it is only useful if you run a quick test with the production image.
+You need to pass at lest password to create such user via ``_AIRFLOW_WWW_USER_PASSWORD_CMD`` or
+``_AIRFLOW_WWW_USER_PASSWORD_CMD`` similarly like for other ``*_CMD`` variables, the content of
+the ``*_CMD`` will be evaluated as shell command and it's output will be set ass password.
+
+User creation will fail if none of the ``PASSWORD`` variables are set - there is no default for
+password for security reasons.
+
++-----------+--------------------------+--------------------------------------------------------------+
+| Parameter | Default                  | Environment variable                                         |
++===========+==========================+==============================================================+
+| username  | admin                    | _AIRFLOW_WWW_USER_USERNAME                                   |
++-----------+--------------------------+--------------------------------------------------------------+
+| password  |                          | _AIRFLOW_WWW_USER_PASSWORD_CMD or _AIRFLOW_WWW_USER_PASSWORD |
++-----------+--------------------------+--------------------------------------------------------------+
+| firstname | Airflow                  | _AIRFLOW_WWW_USER_FIRSTNAME                                  |
++-----------+--------------------------+--------------------------------------------------------------+
+| lastname  | Admin                    | _AIRFLOW_WWW_USER_LASTNAME                                   |
++-----------+--------------------------+--------------------------------------------------------------+
+| email     | airflowadmin@example.com | _AIRFLOW_WWW_USER_EMAIL                                      |
++-----------+--------------------------+--------------------------------------------------------------+
+| role      | Admin                    | _AIRFLOW_WWW_USER_ROLE                                       |
++-----------+--------------------------+--------------------------------------------------------------+
+
+In case the password is specified, the user will be attempted to be created, but the entrypoint will
+not fail if the attempt fails (this accounts for the case that the user is already created).
+
+You can, for example start the webserver in the production image with initializing the internal sqlite
+database and creating an ``admin/admin`` Admin user with the following command:
+
+.. code-block:: bash
+
+  docker run -it -p 8080:8080 \
+    -e _AIRFLOW_DB_UPGRADE="true" \
+    -e _AIRFLOW_WWW_USER_CREATE="true" \
+    -e _AIRFLOW_WWW_USER_PASSWORD="admin" \
+      apache/airflow:master-python3.8 webserver
+
+
+.. code-block:: bash
+
+  docker run -it -p 8080:8080 \
+    -e _AIRFLOW_DB_UPGRADE="true" \
+    -e _AIRFLOW_WWW_USER_CREATE="true" \
+    -e _AIRFLOW_WWW_USER_PASSWORD_CMD="echo admin" \
+      apache/airflow:master-python3.8 webserver
```

Comment:
```suggestion
.. code-block:: bash

  docker run -it -p 8080:8080 \
    -e _AIRFLOW_DB_UPGRADE=true \
    -e _AIRFLOW_WWW_USER_CREATE=true \
    -e _AIRFLOW_WWW_USER_PASSWORD=admin \
      apache/airflow:master-python3.8 webserver


.. code-block:: bash

  docker run -it -p 8080:8080 \
    -e _AIRFLOW_DB_UPGRADE=true \
    -e _AIRFLOW_WWW_USER_CREATE=true \
    -e _AIRFLOW_WWW_USER_PASSWORD_CMD=echo admin \
      apache/airflow:master-python3.8 webserver
```

From the [docs](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file) I would say we remove the `"`. For example I had some issues - not in docker `-e` but in `--env-file` using `"`.

WDYT?

Reply:
https://dev.to/tvanantwerp/don-t-quote-environment-variables-in-docker-268h --> is pretty much the issue I faced, too. But don't know if this also can happen to env variables directly passed into the container.

Reply:
I think this is only issue specific to .env file, the quotes when you pass environment variables via -e switch are not needed when you have no spaces, but they are absolutely necessary when you have spaces (otherwise shell treats them as another parameter). This is a "shelll" expansion standard, not  .env file standard and quoting variables there is really good practice.

For example: 

`-e _AIRFLOW_WWW_USER_PASSWORD_CMD=echo admin`

will set `_AIRFLOW_WWW_USER_PASSWORD_CMD` variable to "echo" and it will try to run "admin" image.

Reply:
This is rather widely used practice.

Reply:
But indeed probably better to quote all arguments in this case

Reply:
Updated to add quotes around whole "VAR=value"